Makefile

```diff
@@ -31,6 +32,14 @@ endif
 test: ## Executes tests
 		$(PHP) ./vendor/bin/phpunit
 
+.PHONY: test-extension
+test-extension: ## Executes tests for extension by first moving src dir
+		sudo sed -i -e 's/;extension=sanitizer.so/extension=sanitizer.so/g' $(INI_FILE)
+		mv src src_
```

Comment:
Bad practice. You may catch a fatal error or segfault during the test execution. It this case the `src` dir will not back